flowmachine/tests/test_cache.py

```diff
@@ -149,18 +132,8 @@ def test_invalidate_cache_multi(flowmachine_connect):
     dl1.invalidate_db_cache()
     assert not dl1.is_stored
     assert not hl1.is_stored
-    in_cache = bool(
-        flowmachine_connect.fetch(
-            f"SELECT * FROM cache.cached WHERE query_id='{dl1.md5}'"
-        )
-    )
-    assert not in_cache
-    in_cache = bool(
-        flowmachine_connect.fetch(
-            f"SELECT * FROM cache.cached WHERE query_id='{hl1.md5}'"
-        )
-    )
-    assert not in_cache
+    assert not cache_table_exists(flowmachine_connect, dl1.md5)
```

Comment:
It just occurred to me that for this test to work properly we assume that the query _was_ previously stored. I know we check this in a different test, just wondering whether it is best to make all tests work independently (in case someone later removes the other test and/or changes something about the functionality)?

If so, we could add
```
assert dl1.is_stored
assert hl1.is_stored
```
above, before `dl1.invalidate_db_cache()`.

Reply:
(A similar observation applies to the other `test_invalidate_...` tests.)